src/lib/openapi/spec/meta-schema.test.ts

```diff
@@ -0,0 +1,33 @@
+import { schemas } from '..';
+import Ajv from 'ajv';
+
+const ajv = new Ajv();
+
+const openApiMetaSchema = {
+    type: 'object',
+    properties: {
+        description: { type: 'string' },
+    },
+    required: ['description'],
+};
+
+describe('OpenAPI schemas meta validation', () => {
+    let countErrors = 0;
+    Object.entries(schemas).forEach(([name, schema]) => {
+        it(`${name} should be valid`, () => {
+            const validateMetaSchema = ajv.compile(openApiMetaSchema);
```

Comment:
I thought about it, but apparently, the way `ajv` works (and correct me if I'm wrong because this is just speculation), is by storing some state inside the `validateMetaSchema` function/object/hybrid? because after you use it as a function on a schema (`validateMetaSchema(schema)`) you use it as an object to check if the previous usage as a function raised an error (`validateMetaSchema.errors?.length`). 

Therefore, I assume it's storing the evaluation result in the function. Because of that, I should either recreate the function/object/hybrid by recompiling it or maybe clean up the errors list (but I don't know if there's more state in that object that needs to be cleaned up).

Reply:
Ok! It's possible, maybe when the function is called it clears the state... :shrug: but it works so :heavy_check_mark: